src/lib/operators/operator_performance_data.hpp

```diff
@@ -26,6 +29,18 @@ struct OperatorPerformanceData : public Noncopyable {
                                 DescriptionMode description_mode = DescriptionMode::SingleLine) const;
 };
 
+struct StagedOperatorPerformanceData : public OperatorPerformanceData {
+  StagedOperatorPerformanceData() : OperatorPerformanceData{}, stage_runtimes{10} {}
```

Comment:
Won't this silently blow up if an operator has more than 10 steps?

Reply:
Ah, that's why you use an array in your example? Would that crash during compilation?

Reply:
If you made 10 a constexpr, you could also do the `static_assert` for the vector. I am using an array so that we don't have a heap allocation for a known, small number of values.

Reply:
What do you mean by static assert?

Reply:
```cpp
static_assert(magic_enum::enum_count<Stages>() < sizeof(stages), "Too many stages");
```